src/template-tags/tickets.php

```diff
@@ -1853,3 +1853,14 @@ function tribe_tickets_get_provider_query_slug() {
 function tec_tickets_tec_events_is_active() : bool {
 	return function_exists( 'tribe_events' );
 }
+
+/**
+ * Return if `The Events Calendar PRO` is active.
+ *
+ * @since TBD
+ *
+ * @return bool True if The Events Calendar PRO is active.
+ */
+function tec_tickets_tec_events_pro_is_active(): bool {
+	return class_exists( 'Tribe__Events__Pro__Main', false );
```

Comment:
Let's not use classes for feature detection anymore.
The upsell should be active if ECP is active and CT1 migration completed.
[I would rely on an action](https://github.com/the-events-calendar/events-pro/blob/534908125864e0bbd29f3fdac7212bff67de21cd/src/Events_Pro/Custom_Tables/V1/Full_Activation_Provider.php#L128) - do not add a template tag, a user-facing API that will have to stay there for a long time, to satisfy an internal requirement.

Reply:
Furthermore, this is not required.
[If FT is active, the CT1 feature is correctly activated in ECP context.](https://github.com/the-events-calendar/event-tickets/blob/cb6043c9df4e0e59847b107e506cc866c3b664c1/src/Tickets/Provider.php#L89-L9)